npm/ng-packs/packages/theme-shared/src/lib/handlers/error.handler.ts

```diff
@@ -132,23 +161,23 @@ export class ErrorHandler {
                 ? this.show404Page()
                 : this.showError(
                     {
-                      key: 'AbpAccount::DefaultErrorMessage404',
+                      key: DEFAULT_ERROR_LOCALIZATIONS.defaultError404.details,
                       defaultValue: DEFAULT_ERROR_MESSAGES.defaultError404.details,
                     },
                     {
-                      key: 'AbpAccount::DefaultErrorMessage404Detail',
+                      key: DEFAULT_ERROR_LOCALIZATIONS.defaultError404.title,
                       defaultValue: DEFAULT_ERROR_MESSAGES.defaultError404.title,
                     },
                   );
               break;
             case 500:
               this.createErrorComponent({
                 title: {
-                  key: 'AbpAccount::500Message',
+                  key: DEFAULT_ERROR_LOCALIZATIONS.defaultError500.title,
                   defaultValue: DEFAULT_ERROR_MESSAGES.defaultError500.title,
                 },
                 details: {
-                  key: 'AbpAccount::InternalServerErrorMessage',
+                  key: DEFAULT_ERROR_LOCALIZATIONS.defaultError500.details,
```

Comment:
This has been changed to `'AbpUi::DefaultErrorMessage'` from `'AbpAccount::InternalServerErrorMessage'`. Is `'AbpAccount::InternalServerErrorMessage'` no longer being used?

Also, I noticed that previous messages had resource name set to `AbpAccount`. New messages have it as `AbpUi`.

Reply:
`'AbpAccount::InternalServerErrorMessage'` still can be used, but `AbpUi` resource name is good choice. Because error.handler provided from ThemeSharedModule. It is not a good approach to use the `AbpAccount`, `AbpIdentity`, etc. resource names in `@abp/ng.theme-shared` package.

Reply:
Got it, then good work!